README.md

```diff
@@ -21,14 +21,13 @@ The hll_map_open_detection folder contains images which are always displayed whe
 4. Install the dependencies using pip in your terminal: `pip install -r requirements.txt`
 5. Install the [obs-websocket plugin](https://obsproject.com/forum/resources/obs-websocket-remote-control-obs-studio-from-websockets.466/) for OBS (Windows Installer works fine)
 6. Start (or restart) OBS (Note that Streamlabs OBS will not work)
-6. Configure the settings in obs_screen_recognition_settings.json:
+6. Configure the settings in `settings.json`:
     - **monitor_to_capture**: If you have multiple monitors this specifies which one should be used.
-    - image_directory: Path to a directory containing images you want to match against
-    - mask_file: Path to an OpenCV mask file which specifies particular areas of the screen to analyze (significantly speeds up the script)
+    - **screen_format**: Your screen format, either `1080p` (if you play in Full HD) or `1440p` (if you play in QUad HD). _Additional formats might be added with your help._
     - **default_scene_name**: OBS scene to open when the script doesn't match an image (e.g. when the map isn't open in HLL)
     - **target_scene_name**: OBS scene to open when the script recognizes an image_directory image in the monitor contents (e.g. when you want the map hidden in HLL)
     - num_features_to_detect: Affects the accuracy and speed of the matching.
     - num_good_matches_required: Affects the false-match frequency
     - show_debug_window: Set to true in order to view the screen that the script is trying to match image_directory images in
-7. Run the script: `python obs_screen_recognition_script.py`, and tweak settings/images if necessary (Enabling `show_debug_window` will show you what the script is seeing).
+7. Run the script: `obs_screen_recognition_script.py`, providing the folder where matching images are stored (currently only one game supported, `hll`) tweak settings/images if necessary (Enable the `--show_debug_window` flag to display what the script is seeing).
```

Comment:
Overall looks good! Could you add an example of the fully valid command here, e.g. `python obs_screen_recognition_script.py ./hll` - always helps to tell people exactly what to type.